.travis.yml

```diff
@@ -106,6 +113,10 @@ install:
   - |
     echo "packages: containers" >> cabal.project
     echo "packages: containers-tests" >> cabal.project
+  - if [ $HCNUMVER -ge 80200 ] ; then echo 'package containers' >> cabal.project ; fi
+  - "if [ $HCNUMVER -ge 80200 ] ; then echo '  ghc-options: -Werror=missing-methods' >> cabal.project ; fi"
+  - if [ $HCNUMVER -ge 80200 ] ; then echo 'package containers-tests' >> cabal.project ; fi
+  - "if [ $HCNUMVER -ge 80200 ] ; then echo '  ghc-options: -Werror=missing-methods' >> cabal.project ; fi"
```

Comment:
Are these newly auto-generated, or did you add them?

Reply:
I simply ran `haskell-ci regenerate` with a more recent version of `haskell-ci`.

Reply:
Cool.